SDWebImage/SDWebImageDownloader.m

```diff
@@ -94,11 +94,32 @@ - (nonnull instancetype)initWithSessionConfiguration:(nullable NSURLSessionConfi
         _downloadQueue.maxConcurrentOperationCount = 6;
         _downloadQueue.name = @"com.hackemist.SDWebImageDownloader";
         _URLOperations = [NSMutableDictionary new];
+        SDHTTPHeadersMutableDictionary *headerDictionary = [SDHTTPHeadersMutableDictionary dictionary];
+        NSString *userAgent = nil;
+#if SD_IOS
```

Comment:
This seems does not handle tvOS, so it's better to use `SD_UIKIT` instead.

Reply:
Nice catch.